public/components/content-list-drawer/content-list-drawer.js

```diff
@@ -379,6 +398,43 @@ export function wfContentListDrawer($rootScope, config, $timeout, $window, conte
                     contentListDrawerController.updateAssigneeUserImage();
                 }
             });
+
+            $scope.setBookSections = function() {
+                var results = wfTagApiService.searchTags($scope.bookSectionQuery, 'newspaper book section');
+                results.then(function (response) {
+                const tags = (response && response.data) || []
+                    return tags.map(function (tag) {
+                        return { item: tag.data };
+                    });
+                }).then(function(candidateBookSections) {
```

Comment:
Any need for this second `then`? The value returned from the previous block doesn't look like another promise.

Reply:
At 440 lines, you can forgive me for my overenthusiasm there. :(

Reply:
Oh, for sure, no criticism – just pedantic linting 😁